public/angular/templates/reports/show.html

```diff
@@ -1,47 +1,242 @@
+    <li class="breadcrumb-item"><a ui-sref="reports" translate>Reports</a></li>
+    <li class="breadcrumb-item active" translate>Report Details</li>
+  </ol>
+</nav>
+
+<ng-include src="'/templates/alerts.html'"></ng-include>
 
-<table class="table detail-table">
-  <tbody>
-    <tr>
-      <td class="detail-key" translate>Authored Time</td>
-      <td class="detail-value">{{ report.authoredAt | aggieDate:'datetime' }}</td>
-    </tr>
-    <tr>
-      <td class="detail-key" translate>Fetched Time</td>
-      <td class="detail-value">{{ report.fetchedAt | aggieDate:'datetime' }}</td>
-    </tr>
-    <tr>
-      <td class="detail-key" translate>Stored Time</td>
-      <td class="detail-value">{{ report.storedAt | aggieDate:'datetime'}}</td>
-    </tr>
-    <tr ng-repeat="source in sources">
-      <td class="detail-key" translate>Sources</td>
-      <td class="detail-value">{{ source.nickname }}</td>
-    </tr>
-    <tr>
-      <td class="detail-key" translate>URL</td>
-      <td class="detail-value"><a ng-href="{{ report.url }}" target="_blank">{{ report.url }}</a></td>
-    </tr>
-    <tr>
-      <td class="detail-key" translate>Author</td>
-      <td class="detail-value">{{ report.author }}</td>
-    </tr>
-    <tr>
-      <td class="detail-key" translate>Tags</td>
-      <td class="detail-value">{{ tagsToString(report.tags) }}</td>
-    </tr>
-    <tr>
-      <td class="detail-key" translate>Content</td>
-      <td class="detail-value" ng-bind-html="report.content">{{ report.content  }}</td>
-    </tr>
-    <tr ng-if="report.metadata.description">
-      <td class="detail-key" translate>Shared Content</th>
-      <td class="detail-value" ng-bind-html="report.metadata.description">{{ report.metadata.description }}</td>
-    </tr>
-  </tbody>
-</table>
+<div class="container-fluid">
+  <div class="row">
+    <div class="col">
+
+    </div>
+    <div class="col-lg-9">
+      <div class="card">
+        <div class="card-header bg-gray">
+          <div class="btn-toolbar justify-content-between" role="toolbar" aria-label="Toolbar with edit confirmations" ng-show="currentUser.can('edit data')">
+            <div class="btn-group">
+              <button type="button" ng-click="$back()" class="btn btn-default mr-2">
+                <i class="fa fa-arrow-left" aria-hidden="true"></i>
+              </button>
+            </div>
+            <div class="right-side">
+              <div class="btn-group mr-4" role="group" aria-label="Flag/Unflag Buttons">
+                <button type="button" class="btn btn-info" ng-click="toggleFlagged(false)">
+                  <span class="mr-2" translate> Un-Flag </span>
+                  <i class="fa fa-flag-o" aria-hidden="true"></i>
+                </button>
+                <button type="button" class="btn btn-danger" ng-click="toggleFlagged(true)">
+                  <span class="mr-2" translate> Flag </span>
+                  <i class="fa fa-flag" aria-hidden="true"></i>
+                </button>
```

Comment:
Flag/unflag seems to have no effect

Reply:
At the time of the PR they were placed there as placeholders because the functionality didn't actually exist on the original aggie. It is now done. See https://github.com/TID-Lab/aggie/commit/0b02c9e7fda490b0e673049feedd484364241a03 for the fix.